index.rst

```diff
@@ -6,20 +6,423 @@
 
+Overview
+========
+
+This document describes the procedure to pump and cool down LATISS from ambient pressure to its operational temperatures. 
+It also outlines how to perform a controlled warm-up in the case of servicing or programmed power cuts. 
+
+Pumping LATISS to vacuum
+========================
+
+Under atypical circumstances, LATISS needs to be warmed up to ambient temperature. 
+Large periods of power outage, severe earthquakes (`and different levels of recovery based on severity <https://tstn-027.lsst.io>`__), or technical intervention may have necessitated warm-up.
+Pumping down, cooling, and powering up LATISS takes at least two days, and must be organized by ticket on the summit calendar and supported by daytime engineering support. 
+Mario Rivera, Craig Lage, and Alysha Shugart are points of contact for understanding and initiating the procedure. 
+
+Getting started
+---------------
+
+AuxTel must be under LOTO or the `E-STOP <https://obs-ops.lsst.io/Daytime-Operations/Auxiliary-Telescope/AuxTel-E-Stop-Procedure/E-Stop-Procecure.html>`__ must be engaged to perform this operation. 
+The safety team is responsible for the the approval of the LOTO procedure, which will be linked on this page once it is affirmed.
+ 
+Start by powering up the instrument cooling electronics, located in the first floor cabinet. 
+It is important to make sure that the electronics are turned on but the outputs are disabled, so no control loops are operating on LATISS until it's under vacuum. 
+These electronics consist of:
+
+-  3 HMP2030 power supplies
+-	1 Keithley 6487 Voltage Source
+-	1 Advantech Uno-1483G-434AE rail controller (not used in the process)
+-	1 Cryocon temperature controller (for CCD sensor)
+
+.. figure:: /_static/power_supply_picometer.png
+   :name: power-supply-picometer
+   :alt: Power supply picometer
+
+   HMP2030 Power Supply (left) and Keithley 6487 Voltage source (right).
+
+.. figure:: /_static/rail_and_cryocon_controller.png
+   :name: rail-and-cryocon-controller
+   :alt: Rail and cryocon controller
+
+   Advantech Rail controller (left) and Cryocon Temperature controller (right).
+
+Setting up the turbo (vacuum) pump
+----------------------------------
+
+The spectrograph needs to work in a good vacuum (~10−7 Torr). For reaching this vacuum, there are two devices involved:
+
+-	Pfeifer Turbopump (to be used from 1 Torr to ~10−6 Torr), complete with vacuum hose, pressure sensor and the sensor’s electronic display
+
+-	Ion pump (to be used from 10−6 Torr onwards)
+
+.. figure:: /_static/turbo_pump_equipment.png
+   :name: turbo-pump-equipment
+   :alt: Turbo pump equipment
+
+   Pfeifer turbopump and accessories.
+
+The Pfeifer turbopump lives in the AuxTel storage container on Calibration Hill. 
+The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container. 
+The cardboard storage box is labeled. 
```

Comment:
Sounds like a bit of a sentence fragment; maybe just add this to the previous sentence?
"The turbo pump, vacuum hose, power supplies, and pressure sensor are stored together on the floor and first shelf of the storage container in a labeled cardboard storage box"

Reply:
👍